salt/modules/aptpkg.py

```diff
@@ -136,7 +138,7 @@ def __init__(self, line, file=None):
             self.uri = ""
             self.line = line
             self.architectures = []
-            self.signed_by = ""
+            self.signedby = ""
```

Comment:
**question** why the rename? Was the previous name buggy? I'm on my phone ATM so can't easily search for uses of this within salt - just want to make sure the rename doesn't catch anything